packages/api/src/schema/datasets/gnomad.js

```diff
@@ -120,17 +150,23 @@ export const fetchGnomadVariant = async (variantId, ctx) => {
   /* eslint-enable no-underscore-dangle */
   const commonData = exomeData || genomeData
 
-  return {
-    // common variant fields
+  const commonVariantFields = {
```

Comment:
May want to call this something different, since "common variant" already has a meaning in genetics. Perhaps `sharedVariantFields`?

Reply:
👍 will change.